packages/sonarwhal/src/lib/utils/npm.ts

```diff
@@ -26,7 +26,7 @@ export const installPackages = (packages: Array<string>): boolean => {
         return true;
     }
 
-    const command: string = `npm install ${packages.join(' ')}${global ? ' -g' : ''}`;
+    const command: string = `npm install ${packages.join(' ')}${global ? ' -g' : ' --save-dev'}`;
```

Comment:
Things are a bit more complicated. Maybe someone installed `sonarwhal` as a regular dependency and thus the package should be installed as a regular dependency. You should check the `package.json` and find where sonarwhal has been installed (`dependencies` or `devDependencies`).

Reply:
@molant Done.